contextualized/regression/lightning_modules.py

```diff
@@ -65,19 +67,32 @@ def __init__(
         self.model_regularizer = model_regularizer
         self.base_y_predictor = base_y_predictor
         self.base_param_predictor = base_param_predictor
-        self._build_metamodel(*args, **kwargs)
+        self._build_metamodel(context_dim, x_dim, y_dim, encoder_type=encoder_type)
```

Comment:
Need to be able to configure width, layers, or other metamodel args than just these.